team-agreements/definition-of-ready/readme.md

```diff
@@ -1,8 +1,30 @@
-# Team Definition Of Ready Agreement
+# Definition of Ready
 
-## Content TODO's
+At the time the development team picks a user story from the top of the backlog to estimate during the Sprint planning, they consider that the user story has a certain level of detail that helps them to give an accurate estimate of completion of the user story within the sprint. This level of information is what makes the story to be Ready to be developed in the sprint.
 
-- Define criteria for a story to be ready to pull into a sprint.
-  - Examples
-    - Has estimate
-    - Team understands implementation/design
+  `In case a user story is not Ready in the beginning of the Sprint it increases the chance that your story will not be done at the end of this print.`
+

```

Comment:
Suggestion for reword to make it a bit more concise:

When the development team picks a user story from the backlog, the user story needs to have enough detail to estimate the work needed to complete the story. If it has enough detail to estimate, it is Ready to be developed in the sprint.

If the user story is not Ready in the beginning of the Sprint, there is a bigger chance that the user story will not be Done by the end of the sprint.

Reply:
Done. Thanks.